.github/workflows/basic-tests.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         pip install -r requirements.txt
         pip install -r companion.txt
         pip install mpi4py
+        pip install ligo.skymap
```

Comment:
Is this needed if you add it to `companion.txt`?

Reply:
Ah, whoops. Obviously not (and now my attempt to understand how to only install this in python3 in github actions syntax was wasted effort ...)